dGame/dComponents/LUPExhibitComponent.h

```diff
@@ -11,7 +11,7 @@
 class LUPExhibitComponent : public Component
 {
 public:
-	inline static const eReplicaComponentType ComponentType = eReplicaComponentType::EXHIBIT;
+	inline static const eReplicaComponentType ComponentType = eReplicaComponentType::LUP_EXHIBIT;
```

Comment:
Like the above, this is a functional change.  See the main comment.

Reply:
I think this was something I noticed in passing, fixed, and forgot, since I did these changes like 3 weeks ago.
Will split out into it's own PR since it is incorrect